utilities/transactions/pessimistic_transaction_db.h

```diff
@@ -155,12 +155,16 @@ class PessimisticTransactionDB : public TransactionDB {
   std::shared_ptr<Logger> info_log_;
   const TransactionDBOptions txn_db_options_;
 
+  static Status FailIfBatchHasTs(const WriteBatch* wb);
+
   void ReinitializeTransaction(
       Transaction* txn, const WriteOptions& write_options,
       const TransactionOptions& txn_options = TransactionOptions());
 
   virtual Status VerifyCFOptions(const ColumnFamilyOptions& cf_options);
 
+  Status FailIfCfEnablesTs(const ColumnFamilyHandle* column_family) const;
```

Comment:
We could only if we remove the following since we need to call `DefaultColumnFamily()` which is non-static.
```
Status FailIfCfEnablesTs(const ColumnFamilyHandle* column_family) const {
  column_family = column_family ? column_family : DefaultColumnFamily();
  ...
}
```